packages/flame_rive/example/assets/spineboy/spineboy.atlas

```diff
@@ -0,0 +1,307 @@
+
+spineboy.png
```

Comment:
Why are these files in the flame_rive directory? And why have you removed skills.riv?

Reply:
my bad,fixed